pyopia/pipeline.py

```diff
@@ -105,16 +113,17 @@ def __init__(self, steps, initial_steps=['initial', 'classifier']):
 
         print('Pipeline ready with these data: ', list(self.data.keys()))
 
-    def run(self):
+    def run(self, filename):
         '''Method for executing the processing pipeline
 
```

Comment:
Could add here something like:

    Args:
        filename (str): file to be processed